src/endless/Analytics.cpp

```diff
@@ -31,7 +32,8 @@ static UINT threadSendRequest(LPVOID pParam)
 		try {
 			CString headers = _T("Content-type: application/x-www-form-urlencoded");
 			CHttpConnection *conn = session.GetHttpConnection(_T(SERVER_NAME), (INTERNET_PORT)SERVER_PORT);
-			CHttpFile *file = conn->OpenRequest(CHttpConnection::HTTP_VERB_POST, _T("collect"));
+			CString path = Analytics::instance()->m_debug ? _T("debug/collect") : _T("collect");
```

Comment:
I'm not wild about accessing the global `Analytics::instance()` from this thread. Pass the debug flag (or the path) as the `pParam` to the thread? This would avoid having to define `m_debug` and declare the thread function a `friend` and so on.